src/components/books/form.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-vars */
+function Form() {
```

Comment:
- [ ] Please, use ES6 best practices, in this case, this could improve being an arrow function. 🦊